### ISSUE TITLE
The 'capture' parameter in the 'purchase' method is not passed

### ISSUE THREAD

**Issue by igor-tv** [hiqdev/omnipay-yandex-kassa#1]
The body of my request looks like this:
```
try {
    $response = $gateway->purchase([
        'amount'    => $this->order->total_in_currency,
        'currency'  => $this->order->currency['code'],
        'capture' => true,
        'returnUrl' => $this->returnUrl(),
        'cancelUrl' => $this->cancelUrl(),
        'transactionId' => uniqid('', true),
        'description' => 'Order #' . $this->order->id,
        'metadata' => array(
            'order_id' => $this->order->id,
        ),
    ])->send();
} catch (Throwable $e) {
    return $result->fail([], $e);
}
```	
Request log on Yandex.Kassa site:
```
{
	"amount": {
		"value": "1100.00",
		"currency": "RUB"
	},
	"description": "Order # 29",
	"confirmation": {
		"type": "redirect",
		"return_url": "http://localhost/..."
	},
	"metadata": {
		"transactionId": "5db9bae0bda7a1.16783758"
	}
}
```
Because the `capture` parameter in the `purchase` method is not passed, the payment accepts the status of `waiting_for_capture`.

If I send a request directly through the Yandex.Checkout SDK PHP, `"capture": true` appears in the logs on the Yandex.Kassa site and the payment immediately becomes `succeeded`.
```
$payment = $client->createPayment(
    array(
        'amount' => array(
            'value' => $this->order->total_in_currency,
            'currency' => $this->order->currency['code'],
        ),
        'confirmation' => array(
            'type' => 'redirect',
            'return_url' => $this->returnUrl(),
        ),
        'capture' => true,
        'description' => 'Order # '. $this->order->id,
    ),
    uniqid('', true)
);
```

**Comment by SilverFire** [comment]
Yes, it's called "2-stage transaction", so the Yandex ensures service providing on your side; otherwise, it returns the payment to a customer

**Comment by SilverFire** [comment]
Then you can [capture()](https://github.com/hiqdev/omnipay-yandex-kassa/blob/master/src/Gateway.php#L89) the payment.

If you don't need this kind of behavior, we will happily accept a pull-request that allows capturing payments automatically. Would you like to do it?

**Comment by igor-tv** [comment]
> If you don't need this kind of behavior, we will happily accept a pull-request that allows capturing payments automatically. Would you like to do it?

Ok, I did it.

**Comment by SilverFire** [comment]
Thank you!
Usually, one pull request per feature is being created, otherwise, it's impossible to run tests against the whole change.

I've merged your pull-requests to a new temporary branch `feature/capture` and created a union pull-request #7 

I will check it later today. Thank you!

**Comment by igor-tv** [comment]
Hello! Have you checked the PR #7? Will it be merged into the master branch?

> Thank you!
> Usually, one pull request per feature is being created, otherwise, it's impossible to run tests against the whole change.
> 
> I've merged your pull-requests to a new temporary branch `feature/capture` and created a union pull-request #7
> 
> I will check it later today. Thank you!

**Comment by SilverFire** [comment]
Merged and released 3.0.3

Thanks for the reminder
